### PR TITLE
108054 label entities - get endpoint

### PR DESCRIPTION
**Pull request by janid1967**
[AB#108054](https://statoildeveloper.visualstudio.com/b6d97460-2e53-47ae-aad4-b41396d90828/_workitems/edit/108054)

### CODE REVIEW

File: src/Equinor.ProCoSys.Completion.Query/LabelQueries/GetLabels/GetLabelsForHostQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        var labelHost =
+            await (from lh in _context.QuerySet<LabelHost>()
+                        .Include(lh => lh.Labels)
+                    where lh.Type ==  request.HostType
+                   select lh)
+                .TagWith($"{nameof(GetLabelsForHostQueryHandler)}.{nameof(Handle)}")
+                .SingleOrDefaultAsync(cancellationToken);
+
+        if (labelHost == null)
```
Comment (jsoi-eq) [inline src/Equinor.ProCoSys.Completion.Query/LabelQueries/GetLabels/GetLabelsForHostQueryHandler.cs:29]: is null

File: src/tests/Equinor.ProCoSys.Completion.Query.Tests/Labels/GetLabels/GetLabelsForHostQueryHandlerTests.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        context.Labels.Add(_labelA);
+        context.Labels.Add(_labelB);
+        context.Labels.Add(_labelC);
+        context.Labels.Add(_labelD);
+        context.LabelHosts.Add(_labelHostWith3Labels);
+        context.LabelHosts.Add(_labelHostWithoutLabels);
+
+        context.SaveChangesAsync().Wait();
+    }
+
```
Comment (jsoi-eq) [inline src/tests/Equinor.ProCoSys.Completion.Query.Tests/Labels/GetLabels/GetLabelsForHostQueryHandlerTests.cs:53]: Add test for when labelhost is not found? (The "labelHost is null" condition)

**Review by jsoi-eq — APPROVED** [review]
